skins/oasis/modules/templates/Footer_Index.php

```diff
@@ -1,5 +1,5 @@
 <footer id="WikiaFooter" class="WikiaFooter <?= $showToolbar ? '' : 'notoolbar' ?>">
-	<div id="BOTTOM_LEADERBOARD"></div>
+	<?= F::app()->renderView('Ad', 'Index', ['slotName' => 'BOTTOM_LEADERBOARD', 'pageTypes' => ['homepage_logged', 'corporate', 'search', 'all_ads']]); ?>
```

Comment:
It will `push` the slot from the beginning which is not OK in bottom leaderboard case.